/**:
  ros__parameters:
    heading_weight: 0.1
    lane_planner:
      trajectory_resolution: 2.0
    vehicle_parameters_node: "/test/vehicle_parameters_node"
    gaussian_smoother:
      standard_deviation: 1.0
      kernel_size: 5
